CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+## Contributing to KubeVirt-Ansible
+
+### Intro
+
+In addition for allowing deployment of KubeVirt via Ansible,
+the KubeVirt-Ansible project is open for any contributions from
+Additional projects which aims to integrate with KubeVirt, such 
+as specific storage or network project.
+
+The integration will be in the form of Ansible roles per component
+which might later be integrated into a main Ansible Playbook Bundle.o
+
+Information on how to contribute will follow in the next sections.
+
+### Contributing new Ansible roles  
+
+Contributing to KubeVirt-Ansible should be as simple as possible. 
+Have a question? Want to discuss something? Want to contribute something? Just open an
+[Issue](https://github.com/kubevirt/kubevirt-ansible/issues) or a [Pull
+Request](https://github.com/kubevirt/kubevirt-ansible/pulls).
+
+## Roles Structure
+
+[TODO]: 
+* Add info on role template and how to role for each supported flow ( K8S,OpenShift, W/ VMs )
+* Add example from existing role for KubeVirt
+
+### Running deployment and tests locally on your laptop
+
+[TODO]:
+* Add info on how to run lago locally to test deployments and run tests
+* Add info on how to use Vagrant to test deployments locally
```

Comment:
I am not sure whether this flow is relevant here.
It is related to developer's environment in kubevirt repository itself.

Reply:
You're right, but it might also be relevant for anyone contributing new roles or APB code and wants to test it locally as well.

What we can do is write it in a separate doc for developer guide and add a link to it from here, as additional info.